features/step_definitions/petition_steps.rb

```diff
@@ -210,6 +210,22 @@
   expect(page).to have_text(constituency)
 end
 
+Then /^I should see my MP/ do
+  @signature = Signature.find_by(email: "womboidian@wimbledon.com",
+                                 postcode: "N11TY",
+                                 name: "Womboid Wibbledon",
+                                 petition_id: @petition.id)
+  expect(page).to have_text(@signature.constituency.mp.name)
+end
+
+Then /^I can click on a link to visit my MP$/ do
+  @signature = Signature.find_by(email: "womboidian@wimbledon.com",
+                                 postcode: "N11TY",
+                                 name: "Womboid Wibbledon",
+                                 petition_id: @petition.id)
+  expect(page).to have_css("a[href*='#{@signature.constituency.mp.url}']")
```

Comment:
We don't need to use ``@signature`` here as it means the variable will "leak" out of the step and might (possibly) interact unexpectedly with other steps that do use ``@signature``.  If the step can be self-contained it's better, so just use a local variable ``signature`` instead of an instance variable ``@signature`` (instance variables in the context of cucumber are effectively global).